common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1408,3 +1417,32 @@ def clean_id(self, padding_char='='):
         return "course_{}".format(
             b32encode(unicode(self.location.course_key)).replace('=', padding_char)
         )
+
+    @property
+    def teams_enabled(self):
```

Comment:
Actually, I don't think we can actually do that check, because this is in `xmodule` and checking the feature flag would require access to the django settings.